pkg/dnsbl/dnsbl.go

```diff
@@ -75,7 +80,7 @@ func (c *Checker) query(ipAddress, bl string, addresses chan<- int) {
 		log.Printf("%v present in %v(%v)", reversedIPAddress, bl, result)
 	}
 	addresses <- len(result)
-	c.Queried++
+	c.queried++
```

Comment:
I'd rather to a `max { len(result), 1 }` over here.

Reply:
This removes the ability to compare the `length` (providers in my list) and the `queried` (providers that could be resolved). To solve both this comment and the previous one we would need an additional metric on the total number of lists the providers hold (total number of possible positives).

But this approach still does not properly present the number of lists we have checked, as those without results return 0 elements.

Reply:
Also, this becomes the exact same as `positive`.